internal/forms/forms.go

```diff
@@ -33,6 +34,7 @@ import (
 
 const fieldValueFalseInXML = "False"
 const txtFileExt = ".txt"
+const formsVersionInfoUrl = "http://www.winlink.org/content/all_standard_templates_folders_one_zip_self_extracting_winlink_express_ver_12142016"
```

Comment:
Any reason for not doing this over HTTPS?

Reply:
Looks like that is resolving now on HTTPS; I think it was not a few months ago.